heron/tools/common/src/python/utils/config.py

```diff
@@ -58,6 +58,12 @@
 ROLE_KEY = "role.required"
 ENVIRON_KEY = "env.required"
 
+# global variabls that can be overriden when necessary.
+# Note that overriden has to happen before the first call
```

Comment:
who overrides them

Reply:
Users who want to customize the tools.

Reply:
I can also add an option to override heron client data files in cli.